.github/workflows/wasmcloud_host.yml

```diff
@@ -268,3 +268,29 @@ jobs:
           token: ${{ secrets.GITHUB_TOKEN }}
           prerelease: false
           draft: false
+  
+  release-charts:
+    if: ${{ startswith(github.ref, 'refs/tags/') || github.event_name == 'workflow_dispatch' }}
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Configure Git
+        run: |
+          git config user.name "$GITHUB_ACTOR"
+          git config user.email "$GITHUB_ACTOR@users.noreply.github.com"
+
+      - name: Install Helm
+        uses: azure/setup-helm@v1
+        with:
+          version: v3.7.1
+
+      - name: Run chart-releaser
+        uses: helm/chart-releaser-action@v1.2.1
```

Comment:
Should we add some chart testing like this?
 https://github.com/marketplace/actions/helm-chart-testing

Reply:
Let me add that as a follow up issue as it is a separate pipeline and concern from this PR.

Reply:
https://github.com/wasmCloud/wasmcloud-otp/issues/288